### DISCUSSION TITLE
Congratulations and question about GraphQL-to-SQL pattern

### DISCUSSION THREAD

**Discussion by frederikhors** [prisma/prisma-client-go#343] [Ideas]
Thanks for your amazing work!

_I'm sorry if there is already something like this discussed here but I can't find it._

What do you think about https://medium.com/@__xuorig__/on-graphql-to-sql-146368a92adc?

**Comment by steebchen** [comment]
Thanks for your feedback!

Regarding your question, I want to first emphasise that Prisma doesn't do anything with GraphQL, since that is up to the user. However, we do have an internal protocol which looks similar to GraphQL which translates simple queries to SQL depending on the exact SQL database (or at some point even _any_ database).

However, we really want to focus that you don't have to think about that, and access data by writing native application code. 

I also see that it's already mentioned in the article – Prisma client already does some data loading for you in order to make some queries faster; and I guess there's a lot more to come in the future.

Let me know if that answer was helpful or if you want to know something specific.
